docs/pages/extensibility/CustomTasks.md

```diff
@@ -136,3 +138,10 @@ module.exports = async function({workspace, dependencies, options}) {
 
 !!! warning
     Depending on your project setup, the UI5 Tooling tends to have lots of open files at the same time during a build. To prevent errors like `EMFILE: too many open files`, we urge custom task implementations to use the [graceful-fs](https://github.com/isaacs/node-graceful-fs#readme) module as a drop-in replacement for the native `fs` module.
+
+
+## Helper Class `TaskUtil`
+
+Custom tasks defining [Specification Version](../Configuration.md#specification-versions) 2.2 or higher have access to an interface of a [TaskUtil](https://sap.github.io/ui5-tooling/api/module-@ui5_builder.tasks.TaskUtil.html) instance.
+
+In this case, a `taskUtil` object is provided as part of the custom task's [parameters](#task-implementation).  Depending on the Specification Version of the custom task, a set of helper functions is available to the implementation. The lowest required Specification Version for every function is listed in the [TaskUtil API reference](https://sap.github.io/ui5-tooling/api/module-@ui5_builder.tasks.TaskUtil.html).
```

Comment:
see my comment above

Reply:
Done

Reply:
Looks like the two terms are still uppercase?

Reply:
The link one reads "Specification Version 2.2" and therefore refers to a specific version. In that case it should be uppercased. Also see https://github.com/SAP/ui5-builder/pull/480#discussion_r466181782

The two occasions in the second paragraph are now lowercase.